client/src/index.js

```diff
@@ -14,11 +14,13 @@ import TextField from '@material-ui/core/TextField';
 import Button from '@material-ui/core/Button';
 import Link from '@material-ui/core/Link';
 import Snackbar from '@material-ui/core/Snackbar';
+var util = require('util');
 
 import { login, signUp, getUID } from './firebase/firebaseAuth.js'
 var util = require('util');
```

Comment:
Please fix this code. I'm getting this error:
![image](https://user-images.githubusercontent.com/36905285/69989172-eafabd00-14f7-11ea-9e7e-80e6bfd3a1a9.png)